main.go

```diff
@@ -29,12 +29,13 @@ func rssHandler(rw http.ResponseWriter, r *http.Request) {
 		Created:     time.Now(),
 	}
 
-	fetch("https://robots.thoughtbot.com/summaries.xml", master)
-	fetch("http://simplecast.fm/podcasts/271/rss", master)
-	fetch("http://simplecast.fm/podcasts/272/rss", master)
-	fetch("http://simplecast.fm/podcasts/282/rss", master)
-	fetch("http://simplecast.fm/podcasts/1088/rss", master)
-	fetch("https://upcase.com/the-weekly-iteration.rss", master)
+	fetch("https://robots.thoughtbot.com/summaries.xml", master, "Giant Robots blog")
```

Comment:
This is all fine, but it might be nice to add some structure. For example, at the top level:

```go
type sourceFeed struct {
  uri  string
  name string
}

// sourceFeeds defines the list of thoughtbot RSS feeds
// to be combined into a master feed.
var sourceFeeds []sourceFeed{
  {uri: "...", name: "..."},
  // ...
}
```

Then the rssHandler can iterate through the sourceFeeds and fetch them.